bgpd/bgp_vty.c

```diff
@@ -7422,7 +7422,7 @@ DEFPY (bgp_condadv_period,
        BGP_STR
        "Conditional advertisement settings\n"
        "Set period to rescan BGP table to check if condition is met\n"
-       "Period between BGP table scans, in seconds; default 60\n")
+       "Period between BGP table scans in seconds; default 60\n")
```

Comment:
let's keep this out of the relevant part of the patch

Reply:
Fixed.